mscore/plugin/api/elements.h

```diff
@@ -110,11 +110,19 @@ class Element : public Ms::PluginAPI::ScoreElement {
        * element. You can use this value to accurately position other elements 
        * related to the same parent.
        *
-       * This value is in spatium units for compatiblity with Element.offsetY.
+       * This value is in spatium units for compatibility with Element.offsetY.
        * \since MuseScore 3.3
        */
       Q_PROPERTY(qreal posY READ posY)
 
+      /**
+       * Bounding box of this element.
+       *
+       * This value is in spatium units for compatibility with other Element positioning properties.
+       * \since MuseScore 3.3
```

Comment:
Could you please change `3.3` to `3.3.1` here? 3.3 version is out already (and a feature freeze wouldn't probably allow to put this PR to that version) but it should be fine to add this property in 3.3.1.

Reply:
Done.